tests/pytests/unit/beacons/test_memusage.py

```diff
@@ -0,0 +1,69 @@
+"""
+    tests.pytests.unit.beacons.test_memusage
+    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+    Memory usage beacon test cases
+"""
+from collections import namedtuple
+
+import pytest
+import salt.beacons.memusage as memusage
+from tests.support.mock import MagicMock, patch
+
+STUB_MEMORY_USAGE = namedtuple(
```

Comment:
We try to avoid module level variables in tests now since this has caused hard to find issues in the test suite if they clash with other testing modules.

Reply:
Proposed resolution: https://github.com/saltstack/salt/pull/59460/files#diff-946e633d532b148d15d99f3dead109c1d0b159c2fca4f2d1692d4b74595bf47aR19-R35